curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/change-styles-based-on-data.md

```diff
@@ -22,10 +22,9 @@ The `style()` method is not limited to setting the `color` - it can be used with
 
 # --instructions--
 
-Add the `style()` method to the code in the editor to set the `color` of the `h2` elements conditionally. Write the callback function so if the data value is less than 20, it returns "red", otherwise it returns "green".
+Add the `style()` method to the code in the editor to set the `color` of the `h2` elements conditionally. Write the callback function so if the data value is less than 20, it returns `red`, otherwise it returns `green`.
 
-**Note**  
-You can use if-else logic, or the ternary operator.
+**Note:** You can use if-else logic, or the ternary operator.
 
 # --hints--
```

Comment:
The names of the colours here (below) are not in backticks. Should they be?

Reply:
Same thing here - I'm more than happy to wrap them in backticks, but should that be done when the test doesn't look for the colour string specifically? 🤔

Reply:
Based on the reply to the other question I posed above, I suggest leaving the color names without backticks for these two instances.